azure-iot-hub/azure/iot/hub/iothub_registry_manager.py

```diff
@@ -663,3 +663,16 @@ def invoke_device_module_method(self, device_id, module_id, direct_method_reques
         return self.protocol.device_method.invoke_module_method(
             device_id, module_id, direct_method_request
         )
+
+    def send_c2d_message(self, device_id, message):
```

Comment:
>send_c2d_message [](start = 8, length = 16)

This AMQP thing feels to be very separate from the registry manager. Is there a reason why this method should be here. Is this a place holder for a future method ?
Is it possible for client to be created from the connection string directly and use the send message or will that mean we have to keep on supporting that client forever ?

Reply:
Yes, it would be separate, but it will be removed as soon as the service implements this in the swagger.

---
In reply to: [379664342](https://github.com/Azure/azure-iot-sdk-python/pull/465#discussion_r379664342) [](ancestors = 379664342)